scripts/compile-eui.js

```diff
@@ -7,7 +7,7 @@ const fs = require('fs');
 const dtsGenerator = require('dts-generator').default;
 
 const IGNORE_BUILD = ['**/webpack.config.js','**/*.d.ts'];
-const IGNORE_TESTS = ['**/*.test.js','**/*.test.ts','**/*.test.tsx','**/*.spec.tsx'];
+const IGNORE_TESTS = ['**/*.test.js','**/*.test.ts','**/*.test.tsx','**/*.spec.tsx', '**/*.test_helper.tsx'];
```

Comment:
we should probably include `.ts` in the future just in case, and we should actually probably be ignoring mock files as well:
```suggestion
const IGNORE_TESTS = ['**/*.test.js','**/*.test.ts','**/*.test.tsx','**/*.spec.tsx', '**/*.test_helper.tsx', '**/*.test_helper.ts', '**/__mocks__/**'];
```